gui/bar_slider.py

```diff
@@ -0,0 +1,180 @@
+from PyQt5.QtCore import Qt, QRect, QSize, pyqtSignal
+from PyQt5.QtWidgets import (QWidget, QSizePolicy)
+from PyQt5.QtGui import QPainter, QBrush, QColor, QPalette
+
+
+def _get_thumb_value(x, canvas_width, right_value):
```

Comment:
underscore prefixes and the like were used in C to avoid symbol conflicts due to global namespace.  Python does not use a global namespace, therefore the underscore is not required.  additionally, should stick to the same style (camel case vs snake_case).

Reply:
agree, fixed